basic_flutter/lib/presentation/screens/employees_list_screen.dart

```diff
@@ -0,0 +1,82 @@
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter/material.dart';
+
+import '/repository/data_provider.dart';
+import '../widgets/list_item.dart';
+import '/business_logic/cubits/response_cubit.dart';
+import '/business_logic/cubits/response_state.dart';
+import '/data/routes.dart';
+
+class EmployeesListScreen extends StatefulWidget {
```

Comment:
Why stateful?

Reply:
@nicotheod  I need the stateful widget for the actions in the appbar. It surely didn't make sense at that time. Should I add a comment for this?